packages/account/src/Components/terms-of-use/terms-of-use.tsx

```diff
@@ -74,12 +77,18 @@ const TermsOfUse = ({
         return localize('Add account');
     };
 
+    const onValuesChange = (values: TTermsOfUseFormProps) => {
+        const current_step = (getCurrentStep?.() || 1) - 1;
```

Comment:
maybe its better to ??
```suggestion
        const current_step = (getCurrentStep?.() ?? 1) - 1;
```

Reply:
@hamza-deriv I got this code line from handleCancel function and added in many places. And in my opinion we have to use ?? in case. Check for this code:

```
const baz = 0 ?? 42;
console.log(baz);
// Expected output: 0

const baz2 = 0 || 42;
console.log(baz2);
// Expected output: 42
```

So if I change it to `||` then in result I can get `current_step = -1` if `getCurrentStep` returns 0.